src/ListMatch.cc

```diff
@@ -406,33 +398,31 @@ SolList Solutions;
 
      if (int(a_list->size() ) != maxContent+1 )
        {
-	 std::cerr << " There is an internal inconsistency in ListMatchupRotShift " << std::endl
-	      << " maxContent  = " << maxContent << std::endl
-	      << " matches->size() = " << a_list->size() << std::endl;
-	 std::cerr << "please store the involved images and contact the developpers" << std::endl;
+          LOGLS_ERROR(_log, "There is an internal inconsistency in ListMatchupRotShift.");
+          LOGLS_ERROR(_log, "maxContent  = " << maxContent);
+	      LOGLS_ERROR(_log, "matches->size() = " << a_list->size());
```

Comment:
Good catch: it was a tab lurking in there. I'm going to give it all a clang-format pass in the near future, which should get any others like this.